src/guc.c

```diff
@@ -138,6 +139,7 @@ bool ts_guc_enable_vectorized_aggregation = true;
 TSDLLEXPORT bool ts_guc_enable_compression_indexscan = false;
 TSDLLEXPORT bool ts_guc_enable_bulk_decompression = true;
 TSDLLEXPORT bool ts_guc_auto_sparse_indexes = true;
+TSDLLEXPORT int ts_guc_default_hypercore_use_access_method = USE_AM_NULL;
```

Comment:
Why not make this an enum?

```suggestion
TSDLLEXPORT UseAccessMethod ts_guc_default_hypercore_use_access_method = USE_AM_NULL;
```